helper/schema/schema.go

```diff
@@ -1453,7 +1453,7 @@ func (m schemaMap) validate(
 	if err != nil {
 		return append(diags, diag.Diagnostic{
 			Severity:      diag.Error,
-			Summary:       "ExactlyOne",
+			Summary:       "Missing required argument",
```

Comment:
This may not be strictly correct, as I believe this would error if _none_ of the arguments were selected, or if _more than one_ of the arguments were selected.

```suggestion
			Summary:       "Invalid number of arguments",
```

Reply:
Ahh yes, true: this can actually be describing two different problems:

* You didn't set any of the arguments in the set.
* You set more than one of the arguments in the set.

I think I'd prefer to change this to "Invalid _combination_ of arguments", because "number" here makes it feel to me like there's the wrong number of arguments in this block _in total_, rather than the wrong number of arguments within this particular set of arguments that isn't directly visible to the user _except_ in this error message. I will change it, though!